### ISSUE TITLE
Generate python for request-response behavior

### ISSUE THREAD

**Issue by jondavid-black** [jondavid-black/AaC#11]
As a developer I want to auto-generate the boilerplate code for a RESTful service from the model so that I can spend my time implementing the business logic.

AC:

1.  Must demonstrate generation of code for the EchoService model
2.  Must use a REST API with a POST endpoint for the echo behavior
3.  Must provide the developer an auto-generated stub to insert the business logic
4.  Must not destroy existing business logic when rerun.
3.  Must generate python code (we'll do other languages later...I promise)
4.  Must be have an automated acceptance test implemented

**Comment by Coffee2Bits** [comment]
Number 6, "Must be have an automated acceptance test implemented" is a little unclear. 

Is this generating an automated test from the behavior acceptance section, like a cucumber test? If we converted the acceptance definition to cucumber tests then we'd need to discuss how to generate the step definition implementation and whether or not we'd supply a standard set of step definitions. I think this story might need to be blocked by another story that focuses on defining and implementing task 6.
